src/components/header/Header.jsx

```diff
@@ -11,13 +14,17 @@ const Header = () => {
 
   return (
     <header className="bg-black header-grid flex p-4">
-      <div className="ml-auto content-end">
-        <ButtonPill text="Log in" onClick={logInOnClick} />
-      </div>
+      {!rootStore.userStore.isLoggedIn ? (
+        <div className="ml-auto content-end">
+          <ButtonPill text="Log in" onClick={logInOnClick} />
+        </div>
+      ) : (
+        <p className="ml-auto content-end text-white">I'm logged in!!!!</p>
+      )}
```

Comment:
We're telling this component to pretty much say "if the user is logged in, show the `I'm logged in!!!!` text. But if the user isn't logged in, show the log in button

Reply:
I think I understand how you did this but that's a very low chance.